File: 프로그래머스/lv1/131705. 삼총사/삼총사.swift
import Foundation

func solution(_ number:[Int]) -> Int {
    var sum = 0
    var result = 0
    for v in number.indices {
        
        
        for i in number.indices where i > v {
            
            for j in number.indices where j > i {
                
                sum = number[v] + number[i] + number[j]
                if sum == 0 {
                    result += 1
                    print(number[v], number[i], number[j] )
                    continue
                }
                
            }
            
        }
        
    }
    return result
}

File: 프로그래머스/lv1/131705. 삼총사/삼총사.py
def solution(number):
    answer = 0
    for i in range(len(number)):
        for j in range(i+1, len(number)):
            for v in range(j+1, len(number)):
                if number[i] + number[j] + number[v] == 0:
                    answer += 1
    return answer